Currently alsa-lib is at 1.1.2, thus accept SND_LIB_MINOR > 0 anyway.

--- a/config.tests/unix/alsa/alsatest.cpp	2015-05-07 16:14:42.000000000 +0200
+++ b/config.tests/unix/alsa/alsatest.cpp	2016-09-11 22:51:18.647973737 +0200
@@ -40,7 +40,7 @@
 ****************************************************************************/
 
 #include <alsa/asoundlib.h>
-#if(!(SND_LIB_MAJOR == 1 && SND_LIB_MINOR == 0 && SND_LIB_SUBMINOR >= 10))
+#if(!(SND_LIB_MAJOR == 1 && (SND_LIB_MINOR > 0 || (SND_LIB_MINOR == 0 && SND_LIB_SUBMINOR >= 10))))
 #error "Alsa version found too old, require >= 1.0.10"
 #endif
 
